test/functional/feature_block_reward_reallocation.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2020 The Dash Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.blocktools import create_block, create_coinbase, get_masternode_payment
+from test_framework.mininode import *
+from test_framework.script import CScript
+from test_framework.test_framework import DashTestFramework
+from test_framework.util import assert_equal, get_bip9_status, hex_str_to_bytes
+
+'''
+feature_block_reward_reallocation.py
+
+Checks block reward reallocation correctness
+
+'''
+
+class BlockRewardReallocationTest(DashTestFramework):
+    def set_test_params(self):
+        self.set_dash_test_params(4, 3, fast_dip3_enforcement=True)
+        self.set_dash_dip8_activation(450)
+
+    def create_test_block(self, version=536870912):
```

Comment:
what's up with this magic number?

Reply:
it's `0x20000000` i.e. not signalling for anything

Reply:
Ahh okay, should we add that info as a comment? like 
```suggestion
    # 536870912 == 0x20000000, i.e. not signalling for anything
    def create_test_block(self, version=536870912):
```